types/three/examples/jsm/webxr/XRHandOculusMeshModel.d.ts

```diff
@@ -0,0 +1,13 @@
+import { Group, Object3D } from './../../../src/Three';
+
+import { HandModelOptions, XRHandModel } from './XRHandModelFactory';
+
+export class XRHandOculusMeshModel {
+    controller: Group;
```

Comment:
At some point it probably makes sense to have a more specific type than `Group` for `controller` here and elsewhere. You don't have to change it it in this PR, just leaving a note.

Reply:
I think this is where `@types/webxr` would come in helpful